tycho-testing-harness/src/main/java/org/eclipse/tycho/test/AbstractTychoIntegrationTest.java

```diff
@@ -143,7 +143,7 @@ protected Verifier getVerifier(String test) throws Exception {
         return getVerifier(test, true);
     }
 
-    protected String getTargetPlatform() {
+    protected static String getTargetPlatform() {
```

Comment:
It does not make much sense to have such a (protected) method when it only delegates to a public other static method as no one can ever override this....

Reply:
Didn't expect its purpose is to be overridden, because it was never done anywhere. But reverted it.